### PR TITLE
blog added

### PR DESCRIPTION
**Pull request by anshulsoni4**
## Related Issue
added one more blog 

Closes: #229

## Description of Changes
added new blog

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots

image

Please provide any necessary screenshots to illustrate the changes made.

### CODE REVIEW

**Comment by anshulsoni4** [comment]
pls reviewwwww .....

File: src/pages/blog/Understanding Cloud Technology: A Comprehensive Guide.md
Comment (Alkaison) [inline src/pages/blog/Understanding Cloud Technology: A Comprehensive Guide.md:1]: Branch has conflicts and I can't see your new blog file.

**Review by Alkaison — CHANGES_REQUESTED** [review]

**Review by anshulsoni4 — COMMENTED** [review]
pls check now

**Comment by Alkaison** [comment]
Push your changes and also resolve the conflict

**Comment by anshulsoni4** [comment]
never mind, i will create a new one

**Comment by anshulsoni4** [comment]
> Push your changes and also resolve the conflict

image


the changes are visible in the code, i dont understand where are the conflicts

**Comment by Alkaison** [comment]
This file `src/pages/blog/Understanding Cloud Technology: A Comprehensive Guide.md` has conflicts.

**Review by anshulsoni4 — COMMENTED** [review]
pls review now, also add the gssoc tag

**Comment by anshulsoni4** [comment]
hey @Alkaison pls review

**Comment by Alkaison** [comment]
- There are no changes to be considered in this PR, check this...

image

**Pull request closed without merging**